src/client/components/ProjectSidebar.tsx

```diff
@@ -389,17 +391,50 @@ const SidebarRow = memo(
           {populationDisplay}
         </Styled.td>
         <Styled.td sx={{ ...style.td, ...style.number, ...{ color: textColor } }}>
-          <span>{deviationDisplay}</span>
-          <span sx={style.deviationIcon}>
-            {(districtId === 0 && absoluteDeviation === 0) ||
-            (districtId !== 0 && absoluteDeviation <= popDeviationThreshold) ? (
-              <Icon name="circle-check-solid" color="#388a64" />
-            ) : intermediateDeviation < 0 ? (
-              <Icon name="arrow-circle-down-solid" color="gray.4" />
-            ) : (
-              <Icon name="arrow-circle-up-solid" color="#000000" />
-            )}
-          </span>
+          <Tooltip
+            placement="top-start"
+            content={
+              districtId !== 0 ? (
+                Math.abs(intermediateDeviation) <= popDeviationThreshold ? (
+                  <div>This district meets the {popDeviation}% population deviation tolerance</div>
+                ) : intermediateDeviation < 0 ? (
+                  <div>
+                    Add {Math.floor(intermediateDeviation + popDeviationThreshold).toLocaleString()}{" "}
```

Comment:
We don't want a negative number here:
```suggestion
                    Add {Math.floor(Math.abs(intermediateDeviation) + popDeviationThreshold).toLocaleString()}{" "}
```